public/js/statistics-visited.js

```diff
@@ -0,0 +1,100 @@
+// Create root element
+var root = am5.Root.new("chartdiv");
```

Comment:
same as the previous comment. We can review and replace the variables to use const